circuits/package.json

```diff
@@ -12,7 +13,7 @@
     "circom_tester": "^0.0.20",
     "circomlib": "^2.0.5",
     "mocha": "^10.2.0",
-    "snarkjs": "^0.7.2"
+    "snarkjs": "^0.7.0"
```

Comment:
How come you downgraded `snarkjs`? We're using 0.7.2 on the frontend and I think its a good idea to use same versions, maybe we put it in the root even to keep a single version